src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -99,32 +99,56 @@
                       Business or Corporation is Unregistered in B.C.
                     </a>
 
-                    <v-card v-if="wasBusinessSelectedFromLookup" outlined class="message-box rounded-0 mt-6">
-                      <p>
-                        <strong>Important:</strong> If the addresses shown below are out of date, you may
-                        update them here. The updates are applicable only to this registration.
-                      </p>
-                    </v-card>
-
-                    <template v-else>
-                      <p class="info-text mt-6 pt-0 mb-0">
-                        To add a registered B.C. business or corporation as
-                        {{ isProprietor ? 'the Proprietor' : 'a Partner' }},
-                        enter the name or incorporation number.
-                      </p>
-
-                      <p class="info-text mt-6 pt-0 mb-0">
-                        If you are adding a company that is not legally required to register in B.C.
-                        such as a bank or a railway, use the manual entry form. All other types of
-                        businesses cannot be {{ isProprietor ? 'the Proprietor' : 'a partner' }}.
-                      </p>
-
+                    <div v-if="wasBusinessSelectedFromLookup">
```

Comment:
If this div (which is rendered) does nothing except hold logic then I suggest changing it to a template (which isn't rendered).